modules/ROOT/pages/type-definitions/types.adoc

```diff
@@ -1,5 +1,7 @@
 [[type-definitions-types]]
 = Types
+:page-aliases: type-definitions/types/index.adoc, type-definitions/types/scalar.adoc, type-definitions/types/temporal.adoc, \
```

Comment:
I genuinely don't know if we can use all these links to this same page. Did it work fine when you tested locally?